File: problemas/iniciante/2763/2763.exs
partes = IO.gets("") |> String.trim |> String.split(".")
partes2 = partes |> Enum.at(2) |> String.split("-")

IO.puts(Enum.at(partes, 0))
IO.puts(Enum.at(partes, 1))
IO.puts(Enum.at(partes2, 0))
IO.puts(Enum.at(partes2, 1))

File: problemas/iniciante/2763/2763.php
<?php
$partes = explode(".", trim(fgets(STDIN)));
$partes2 = explode("-", $partes[2]);

echo $partes[0] . PHP_EOL;
echo $partes[1] . PHP_EOL;
echo $partes2[0] . PHP_EOL;
echo $partes2[1] . PHP_EOL;
?>